adot/utils/expected-templates/python.json

```diff
@@ -9,7 +9,27 @@
     "name":"hello-lambda-python.*"
   },
   {
-    "name":"hello-lambda-python.*"
+    "name":"hello-lambda-python.*",
+    "subsegments": [
+      {
+          "name": "HTTP GET"
+      },
+      {
+          "name": "S3",
+          "aws": {
+            "operation": "ListBuckets"
+          },
+          "metadata": {
+            "default": {
+              "aws.service": "s3",
```

Comment:
Similar to https://github.com/aws-observability/aws-otel-test-framework/pull/383#discussion_r742113179, not sure if this is needed. Maybe it is used by the AWS X-Ray backend to get the `"origin":"AWS::S3",` value on the S3 inferred span?